src/core/inlines.js

```diff
@@ -258,9 +258,11 @@ export function run(conf) {
 
   // PRE-PROCESSING
   /** @type {NodeListOf<HTMLElement>} */
-  const abbrs = document.querySelectorAll("abbr[title]");
-  for (const abbr of abbrs) {
-    abbrMap.set(abbr.textContent, abbr.title);
+  const abbrElements = document.querySelectorAll("abbr[title]:not(.exclude)");
```

Comment:
We should document the `exclude` class while we're here.
It's also used with WebIDL blocks apparently: https://github.com/w3c/respec/blob/77ab0c7507d9960c50745dde954a87e199e2be81/tests/spec/core/idl-index-spec.js#L62

Reply:
Correct. We might need to update our PR template so we don't forget to document things.